src/Akka.Monitoring/AkkaMonitoringLogger.cs

```diff
@@ -8,12 +8,38 @@ namespace Akka.Monitoring
     /// Logging implementation used to report key events, such as <see cref="DeadLetter"/> and <see cref="UnhandledMessage"/> instances
     /// back to an appropriate monitoring service
     /// </summary>
-    public class AkkaMonitoringLogger : TypedActor,
-        IHandle<DeadLetter>, IHandle<UnhandledMessage>,
-        IHandle<LogEvent>
+    public class AkkaMonitoringLogger : ActorBase
     {
         public const string LoggerName = "AkkaMonitoringLogger";
 
+        private readonly ActorMonitor _monitor = ActorMonitoringExtension.Monitors(Context.System);
```

Comment:
I've extracted monitor to separate field, so we don't need to resolve it on every incoming event.